tests/unit/models/base-channel-test.js

```diff
@@ -70,6 +70,28 @@ module('Unit | Model | base-channel', function(hooks) {
     assert.equal(channel.get('sortedMessages').length, 0);
   });
 
+  //
+  // sortedUserList
+  //
+
+  test('#sortedUserList returns an empty array if there are no users', function(assert) {
+    const channel = BaseChannel.create();
+
+    assert.equal(typeof channel.get('sortedUserList'), 'object');
```

Comment:
This would be true for any object, but you can check for an array using [`Array.isArray`](https://ecma-international.org/ecma-262/5.1/#sec-15.4.3.2).